src/borg/archiver.py

```diff
@@ -3132,8 +3133,12 @@ def define_borg_mount(parser):
                               help='do not store ctime into archive')
         fs_group.add_argument('--nobirthtime', dest='nobirthtime', action='store_true',
                               help='do not store birthtime (creation date) into archive')
+        # --nobsdflags is misleading and deprecated.
+        # use --noflags as a replacement.
```

Comment:
as this is now in help text, maybe we do not need the comment at all.

Reply:
Removed and rebased everything into one commit.